src/main/java/cs2103t/ip/duke/Ui.java

```diff
@@ -1,79 +1,80 @@
 package cs2103t.ip.duke;
 
+import javafx.application.Platform;
+
 public class Ui {
 
     private final String LINE = "_______________________________\n";
 
-    public void showLine() {
-        System.out.println(LINE);
+    public String showLine() {
+        return LINE;
     }
 
-    public void showWelcome() {
-        System.out.println(LINE +
+    public String showWelcome() {
+        return LINE +
                 "Hello I'm Duke\n" +
                 "What can I do for you?\n" +
-                LINE
-        );
+                LINE;
     }
 
-    public void showBye() {
-        System.out.println(LINE +
-                "Bye. Hope to see you again soon!\n" +
-                LINE);
+    public String showBye() {
+        return LINE +
+                "Bye. Hope to see you again soon! (Closing window in 3 seconds) \n" +
+                LINE;
     }
 
-    public void showMark(Tasklist list, int taskNum) {
-        System.out.println(LINE +
+    public String showMark(Tasklist list, int taskNum) {
+        return LINE +
                 "Nice! I've marked this task as done: \n" +
                 list.getTasks().get(taskNum - 1).getStatusIcon() +
-                list.getTasks().get(taskNum - 1).description + "\n" + LINE);
+                list.getTasks().get(taskNum - 1).description + "\n" + LINE;
     }
 
-    public void showUnmark(Tasklist list, int taskNum) {
-        System.out.println(LINE +
+    public String showUnmark(Tasklist list, int taskNum) {
+        return LINE +
                 "OK, I've marked this task as not done yet: \n" +
                 list.getTasks().get(taskNum - 1).getStatusIcon() +
-                list.getTasks().get(taskNum - 1).description + "\n" + LINE);
+                list.getTasks().get(taskNum - 1).description + "\n" + LINE;
```

Comment:
Maybe this `LINE` can be placed in a separate line in order to stay consistent?
```suggestion
                list.getTasks().get(taskNum - 1).description + "\n" +
                LINE;
```